inject-kotlin/src/main/kotlin/io/micronaut/kotlin/processing/visitor/AbstractKotlinElement.kt

```diff
@@ -605,6 +610,8 @@ internal abstract class AbstractKotlinElement<T : KotlinNativeElement>(
     }
 
     companion object {
+        val extraOpenAnnotations = System.getProperty("kotlin.allopen.annotations")?.split(",")?.toTypedArray()
```

Comment:
Can we get in as a part of the plugin configuration maybe too?

Reply:
Probably yes, maybe even have it automatically if the allopen plugin exists @melix, right?
Let's first see that it's working.